osbuild/meta.py

```diff
@@ -410,6 +410,42 @@ def _parse_caps(cls, _klass, _name, node):
 
     @classmethod
     def load(cls, root, klass, name) -> Optional["ModuleInfo"]:
+        base = cls.MODULES.get(klass)
+        if not base:
+            raise ValueError(f"Unsupported type: {klass}")
+        path = os.path.join(root, base, name)
+
+        try:
+            return cls._load_from_json(path, klass, name)
+        except FileNotFoundError:
+            # should we print a deprecation warning here?
```

Comment:
I don't think so, not until we convert all (current) stages to use `-meta.json`.